### PR TITLE
Add debian packages for fr_ch and tr and fix some build config errors

### PR DESCRIPTION
**Pull request by mariogrip**

### CODE REVIEW

File: plugins/fr_ch/src/src.pro [hunk 1]
```diff
@@ -20,25 +20,25 @@ TARGET          = $$qtLibraryTarget(frplugin)
 EXAMPLE_FILES = frenchplugin.json
 
 # generate database for presage:
-PLUGIN_INSTALL_PATH = $${UBUNTU_KEYBOARD_LIB_DIR}/fr/
```
Comment (NeoTheThird) [inline plugins/fr_ch/src/src.pro:23]: Are you removing french here?

Comment (mariogrip) [reply plugins/fr_ch/src/src.pro:23]: no, without this it will place it in the fr folder, we want them to be where the qml files are

Comment (NeoTheThird) [reply plugins/fr_ch/src/src.pro:23]: ah, sorry, nvm

**Review by NeoTheThird — APPROVED** [review]
lgtm

**Comment by NeoTheThird** [comment]
Okay, jenkins says no. What's going on here?

**Review by NeoTheThird — APPROVED** [review]
there we go

**Comment by NeoTheThird** [comment]
We should probably squash this...

**Pull request merged**